framework/common/nng/broker.cpp

```diff
@@ -18,7 +18,8 @@ namespace mapf {
 
 Broker::Broker() : frontend_(new msglib_socket), backend_(new msglib_socket)
 {
-    int rc = nng_pub0_open_raw(&backend_->sd_);
+    context_ = nullptr; // TODO: implement
```

Comment:
IMO it's more appropriate to do this in the initialiser list than in the function body.

The TODO is wrong - there simply is no context needed in the nng case. This stuff is implemented in a bit an awkward way: there are two implementations, but they use the same header file. For that reason the context_ is a void pointer instead of something useful.